olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/DataRequest.h

```diff
@@ -33,11 +33,11 @@ namespace read {
 
```

Comment:
Can you maybe also adapt DataRequest to coding style and remove layer_id_ as it is not used anymore.

Reply:
Done